README.md

```diff
@@ -25,5 +25,7 @@ Best place to start is our [contribution guide](/CONTRIBUTING.md).
 
 ---
 
+This project was based on [JSON Resume](https://github.com/jsonresume)
```

Comment:
Not sure on this but do we not write "is" instead of "was"?

Reply:
Good catch. "is" feels more natural. As a native speaker David can probably add his thoughts.